scripts/dappnode_install_pre.sh

```diff
@@ -4,19 +4,8 @@
 
 DAPPNODE_DIR="/usr/src/dappnode"
 LOGS_DIR="$DAPPNODE_DIR/logs"
-DOCKER_PKG="docker-ce_20.10.6~3-0~debian-bullseye_amd64.deb"
-DOCKER_CLI_PKG="docker-ce-cli_20.10.6~3-0~debian-bullseye_amd64.deb"
-CONTAINERD_PKG="containerd.io_1.4.4-1_amd64.deb"
-DOCKER_REPO="https://download.docker.com/linux/debian/dists/bullseye/pool/stable/amd64"
-DOCKER_PATH="${DAPPNODE_DIR}/bin/docker/${DOCKER_PKG}"
-DOCKER_CLI_PATH="${DAPPNODE_DIR}/bin/docker/${DOCKER_CLI_PKG}"
-CONTAINERD_PATH="${DAPPNODE_DIR}/bin/docker/${CONTAINERD_PKG}"
-DCMP_PATH="/usr/local/bin/docker-compose"
-DOCKER_URL="${DOCKER_REPO}/${DOCKER_PKG}"
-DOCKER_CLI_URL="${DOCKER_REPO}/${DOCKER_CLI_PKG}"
-CONTAINERD_URL="${DOCKER_REPO}/${CONTAINERD_PKG}"
-DCMP_URL="https://github.com/docker/compose/releases/download/v2.5.0/docker-compose-linux-x86_64"
-WGET="wget -q --show-progress --progress=bar:force"
+lsb_dist="$(. /etc/os-release && echo "$ID")"
```

Comment:
What if this commands returns a non-expected value, like someone is trying to install dappnode on a non-debian based machine. There should be kind of a check for the value output. But maybe its for a different issue

Reply:
You're right, we should definitely check for that somewhere! Probably right place for that is at the very beginning.